satellite/metainfo/loop.go

```diff
@@ -58,6 +60,8 @@ type LoopConfig struct {
 }
 
 // Loop is a metainfo loop service.
+//
+// architecture: Service
```

Comment:
I'm not clear that this is a service, can you explain to me why it's?

Reply:
I'm kind of going back and forth with some of the things whether they are chores or services. And I think we need a clarification what a service/chore is.

Effectively we have three properties:

A. concurrent-safety: e.g. it would be dangerous to run multiple tallys at the same time without coordination
B. interval: e.g. some things require to happen in an interval
C. things that can be depended on: e.g. metainfo.Loop

We have different combinations of these:

1. concurrent-safe, non-interval, can be depended: e.g. overlay.Service
2. concurrent-safe, interval based, can be depended: e.g. metainfo.Loop
3. concurrent-safe, interval based, should not be depended on: e.g. audit.Worker
4. non-concurrent-safe, interval based, should not be depended on: e.g. audit.Chore

Currently "Service" means it can be depended on and it is concurrent-safe.

Interval-based and should not be depended on, are considered Chores. Currently there's a distinction for "Workers" to indicate, that they are concurrent-safe and usually do work based on a queue.

I'm not certain this organization makes sense, or whether everything is correct based on these criteria.

Reply:
I'm a bit worry that this may be hard to understand, however, I'm fine to go ahead with this and tune the definition through the time.

Reply:
I think I agree with the definition of `Service`. For me its something that I can depend on and get some results on demand (concurrent-safe or not). I'm not sure if `concurrent-safe` property fits `Chores` because my understanding is that we are starting them, they can depend on services and they are doing some repetitive work. We don't care what's happening internally. On the other hand, `concurrent-safe` may indicate if it's possible to start one chore several times independently.

Reply:
Anyways, I added a discussion point to architecture meeting, so we can flesh it out a bit more.